nominatim/tools/country_info.py

```diff
@@ -74,8 +79,21 @@ def setup_country_tables(dsn, sql_dir, ignore_partitions=False):
             lang = props['languages'][0] if len(props['languages']) == 1 else None
             params.append((ccode, partition, lang))
 
+            name = add_prefix_to_keys(props.get('names').get('name'), 'name:')
+            name = json.dumps(name, ensure_ascii=False, separators=(', ', '=>'))
+            country_names_data += ccode + '\t' + name[1:-1] + '\n'
     with connect(dsn) as conn:
         with conn.cursor() as cur:
+            cur.execute(
+                """ CREATE TABLE public.country_name (
+                        country_code character varying(2),
+                        name public.hstore,
+                        derived_name public.hstore,
+                        country_default_language_code text,
+                        partition integer
+                    ); """)
+            data = StringIO(country_names_data)
+            cur.copy_from(data, 'country_name', columns=('country_code', 'name'))
```

Comment:
Bonus points, if you merge the query below in here. The additional update was only necessary because the original content came from an external file. Now that you construct the table here from scratch, all the columns can be filled in one go.

Also: the copy is okay in principle. However, given that the table is relatively small, a mass-INSERT would just be fine as well. The advantage of the insert is that you don't have to correctly format and escape the fields yourself, i.e. the whole json.dumps is not necessary.